src/transformers/models/hubert/modeling_hubert.py

```diff
@@ -490,17 +491,19 @@ def forward(
         src_len = key_states.size(1)
         attn_weights = torch.bmm(query_states, key_states.transpose(1, 2))
 
-        if attn_weights.size() != (bsz * self.num_heads, tgt_len, src_len):
-            raise ValueError(
-                f"Attention weights should be of size {(bsz * self.num_heads, tgt_len, src_len)}, but is"
-                f" {attn_weights.size()}"
-            )
-
-        if attention_mask is not None:
-            if attention_mask.size() != (bsz, 1, tgt_len, src_len):
+        if not is_torch_fx_proxy(attn_weights):
```

Comment:
let's do the check a lot earlier, we can check this in the model's forward only once!

Reply:
I am not sure about this place. This block contain

```
            attn_weights = attn_weights.view(bsz, self.num_heads, tgt_len, src_len) + attention_mask
            attn_weights = attn_weights.view(bsz * self.num_heads, tgt_len, src_len)
```
and it should be run no matter if we are in fx proxy or not